src/components/HeroBanner/HeroBanner.js

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+
+import "./HeroBanner.css";
+
+const HeroBanner = ({ title, subtitle, children }) => {
+  return (
+    <section className="HeroBanner">
+      <div className="HeroBanner__text">
+        {!!title && <h1>{title}</h1>}
+        {!!subtitle && <p>{subtitle}</p>}
+      </div>
+      <div class="HeroBanner__content">
+        {children}
+      </div>
+    </section>
+  );
+};
+
+HeroBanner.propTypes = {
+  bgColor: PropTypes.node,
```

Comment:
Are You sure that this is node (html element) ? Maybe it's string. If label could be node write something like this
label: PropTypes.oneOfType([ PropTypes.string, PropTypes.node ]),